spec/registry.md

```diff
@@ -384,6 +322,128 @@ its _resolved value_ contains the implementation-defined integer value
 of the _operand_ of the annotated _expression_,
 together with the resolved options' values.
 
+### The `:currency` function
+
+The function `:currency` is an OPTIONAL selector and formatter for currency values, 
+which are a specialized form of numeric selection and formatting.
+
+#### `:currency` Operands
+
+The function `:currency` requires a [Currency Operand](#currency-operands) as its _operand_
+or a [Number Operand](#number-operands), if used with the _option_ `currency`.
+
+#### Options
+
+Some options do not have default values defined in this specification.
+The defaults for these options are implementation-dependent.
+In general, the default values for such options depend on the locale, 
+the currency,
+the value of other options, or all of these.
+
+Fraction digits for currency values behave differently than for other numeric formatters.
+The number of fraction digits displayed is usually set by the currency used.
+For example, USD uses 2 fraction digits, while JPY uses none.
+Setting `fractionDigits` to `none` makes the display and selection work similar to `:integer`.
```

Comment:
Given this, how would `fractionDigits=none` be different from `fractionDigits=0`? Why is a name needed in addition to the numerical value?

Reply:
My thinking is that `fractionDigits=auto` is the high-runner case and `fractionDigits=none` is in second place. It's semantically different than setting a specific number of digits?

There might be rounding mode differences, I suppose: I didn't deal with rounding mode. Otherwise, there is no difference.

Reply:
I would rather strongly prefer dropping `none`.

Reply:
It was removed.